### ISSUE TITLE
RedirectException thrown on every redirect

### ISSUE THREAD

**Issue by jessedobbelaere** [forkcms/forkcms#1101]
I just pulled/merged the latest changes of Fork CMS but noticed that every time a redirect happens, I get an exception:

    Redirect
    500 Internal Server Error - RedirectException
    
	Common\Exception\RedirectException: Redirect
	    at n/a
	        in /home/vagrant/apps/default/mywebsitenamehere/meeting/src/Backend/Core/Engine/Base/Object.php line 152

	    at Backend\Core\Engine\Base\Object->redirect('/private/nl/form_builder/index?token=1cb0fd2agm&report=edited&var=Overnachting&highlight=row-3')
	        in /home/vagrant/apps/default/mywebsitenamehere/meeting/src/Backend/Modules/FormBuilder/Actions/Edit.php line 321

	    at Backend\Modules\FormBuilder\Actions\Edit->validateForm()
	        in /home/vagrant/apps/default/mywebsitenamehere/meeting/src/Backend/Modules/FormBuilder/Actions/Edit.php line 40

	    at Backend\Modules\FormBuilder\Actions\Edit->execute()
	        in /home/vagrant/apps/default/mywebsitenamehere/meeting/src/Backend/Core/Engine/Action.php line 102

So basically in [this block of code](https://github.com/forkcms/forkcms/blob/master/src/Backend/Core/Engine/Base/Object.php#L147-L152): 

	public function redirect($URL, $code = 302)
	{
	    $response = new RedirectResponse($URL, $code);
	    throw new RedirectException('Redirect', $response);
	}

Shouldn't there be a `$response->send();` in there? Then everything seems to redirects fine. I could do a PR to fix this.

**Comment by WouterSioen** [comment]
This is correct, it should be catched in the app/routing.php file. Can you check if it gets in the catch(RedirectException $ex) block?

**Comment by jessedobbelaere** [comment]
I don't think so. When `throw new RedirectException` is executed, I see that it goes to HttpKernel, which then executes `$this->handleException` and so on. But I set up a breakpoint inside the catch, and it doesn't seem to get there.

**Comment by WouterSioen** [comment]
how did you update your code? With git? Or manually? It seems like you updated the src/ folder but not the app/ folder. The app/routing.php files contained important changes.

**Comment by jessedobbelaere** [comment]
With a git pull/merge. Hmm I'll try if I see this error in a clean install first.

**Comment by WouterSioen** [comment]
Also try to do a new composer install, this doesn't only update your dependencies, but also clears your cache.

**Comment by jessedobbelaere** [comment]
Hmm but I already cleaned up all the cache in app, backend and frontend, and did a composer install after I pulled/updated forkcms. I just installed a new Fork CMS installation (from github) and I also get the redirect error: when I login, when I edit a module etc.

**Comment by WouterSioen** [comment]
I found the issue and fixed it!

**Comment by jessedobbelaere** [comment]
Great! :+1:
